modules/openapi-generator/src/main/java/org/openapitools/codegen/DefaultCodegen.java

```diff
@@ -470,6 +470,13 @@ public Map<String, ModelsMap> postProcessAllModels(Map<String, ModelsMap> objs)
         for (Map.Entry<String, ModelsMap> entry : objs.entrySet()) {
             CodegenModel model = ModelUtils.getModelByName(entry.getKey(), objs);
 
+            if (model.discriminator != null && model.discriminator.getMappedModels() != null) {
```

Comment:
can you please add some comments explaining what this code block does?

cc @OpenAPITools/generator-core-team

Reply:
Done